modules/olm-pruning-index-image.adoc

```diff
@@ -24,35 +26,46 @@ endif::[]
 [id="olm-pruning-index-image_{context}"]
 = Pruning an index image
 
-An index image, based on the Operator Bundle Format, is a containerized snapshot
-of an Operator catalog. You can prune an index of all but a specified list of
-packages, creating a copy of the source index containing only the Operators
-that you want.
+An index image, based on the Operator Bundle Format, is a containerized snapshot of an Operator catalog. You can prune an index of all but a specified list of packages, creating a copy of the source index containing only the Operators that you want.
 
-ifeval::["{context}" == "olm-restricted-networks"]
+ifeval::["{context}" != "olm-managing-custom-catalogs"]
 When configuring Operator Lifecycle Manager (OLM) to use mirrored content on restricted network {product-title} clusters, use this pruning method if you want to only mirror a subset of Operators from the default catalogs.
+
+For this example, this procedure assumes the target registry is a previously created mirror registry that is accessible by both your cluster and a workstation with unrestricted network access. This example also shows mirroring the default `{catalog-name}` catalog, however it can be repeated for the other default catalogs, as well as custom catalogs.
 endif::[]
 
 .Prerequisites
 
+ifeval::["{context}" != "olm-managing-custom-catalogs"]
 * Workstation with unrestricted network access
+endif::[]
 * `podman` version 1.4.4+
 * link:https://github.com/fullstorydev/grpcurl[`grpcurl`]
 * `opm` version 1.12.3+
-* Access to mirror registry that supports
+* Access to a registry that supports
 link:https://docs.docker.com/registry/spec/manifest-v2-2/[Docker v2-2]
-* If you are working with private registries, set the `REG_CREDS` environment
-variable to the file path of your registry credentials for use in later steps.
-For example, for the `podman` CLI:
+
+.Procedure
+
+ifndef::openshift-origin[]
+ifeval::["{context}" != "olm-managing-custom-catalogs"]
+. Authenticate with `registry.redhat.io`:
 +
 [source,terminal]
 ----
-$ REG_CREDS=${XDG_RUNTIME_DIR}/containers/auth.json
+$ podman login registry.redhat.io
 ----
+endif::[]
+endif::[]
 
-.Procedure
+. Authenticate with your target registry:
++
+[source,terminal]
+----
+$ podman login <target_registry>
+----
 
-. On your workstation with unrestricted network access, run the source index image that you want to prune in a container. For example:
+. Run the source index image that you want to prune in a container. For example:
```

Comment:
I think it might be helpful to explain all the modifiable bits of the command, perhaps like:

```
. Run the source index image that you want to prune in a container:
+
[source,terminal,subs="attributes+"]
----
$ podman run <with all of the items you can change in variables> <1> <n>
----
<1> Explanations
<n> Explanations
+
For example:
+
[source,terminal,subs="attributes+"]
----
$ podman run -p50051:50051 \
    -it {index-image-pullspec}
----
...
```

Reply:
I see some other un-explained `podman run` commands throughout our modules; I'll dig into this in a follow-up PR and try to address them together.